### ISSUE TITLE
Sleek fails to open with error `GetVSyncParametersIfAvailable() failed for 1 times!`

### ISSUE THREAD

**Issue by jokuNimi** [ransome1/sleek#359]
**Is it an actual bug?**
Yes. No features are usable.

**Did you check if the bug has already been reported?**
Yes, I tried

**Describe the bug**
After launching sleek, I get errors in terminal:
```
~ % sleek
Persisted data located at: /home/valtteri/.config/sleek/user-preferences.json
Success: Translations loaded for: en
libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
[55269:0405/103236.539623:ERROR:sandbox_linux.cc(377)] InitializeSandbox() called with multiple threads in process gpu-process.
Success: Window events setup
Success: Renderer window created
[55269:0405/103243.616359:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 1 times!
[55269:0405/103246.423549:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 2 times!
[55269:0405/103315.231870:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 3 times!
```
and just a white screen where sleek should be (see screenshots section)

**To Reproduce**
I'm not sure how. The problem appeared suddenly, yesterday if I remember correctly. It doesn't seem to be related to a sleek upgrade, but is most likely related to some other package upgrade.

**Do you see any error entries in sleeks developer tools?**
The given shortcut doesn't open developer tools.

**Expected behavior**
The application should launch as normal.

**Screenshots**
![2022-04-05T10:37:59,891624688+03:00](https://user-images.githubusercontent.com/8011580/161703136-9ce56abf-a4b7-41ed-b883-63991e0b92b8.png)

**Desktop (please complete the following information):**
 - OS: Arch Linux, fully updated, kernel version `5.17.1-arch1-1`.
 - Version of sleek: `1.1.7-1`
 - Source: Arch User Repository

**Comment by ransome1** [comment]
@jokuNimi thanks for reporting this. Before I look into it, could you please check if the AppImage of the latest pre release produces the same behavior on your system?

https://github.com/ransome1/sleek/releases/tag/v1.1.8-rc.4

**Comment by jokuNimi** [comment]
I checked: AppImages `1.1.8-rc.4`, `1.1.7` and `1.1.6` work correctly, meaning they don't produce the same behavior. 

I now also tried reinstalling sleek from AUR; that didn't help.

**Comment by ransome1** [comment]
@jokuNimi I'm seeing the `GetVSyncParametersIfAvailable() failed for 1 times!` error on my Arch too, but sleek installed from aur works fine. So I don't think this is the problem.

Let's start with excluding issues with the preferences file. It's located in your home folder: `/home/USERNAME/.config/sleek/user-preferences.json`.

Could you please backup that file and then delete the original one? Let's see if you can start sleek like this. Also if this is the problem, would you mind sharing your prefrences files here?

**Comment by jokuNimi** [comment]
@ransome1 I backed up and then deleted `user-preferences.json`, and it didn't fix the issue. 

An idea: the error `[11770:0405/124931.800272:ERROR:sandbox_linux.cc(377)] InitializeSandbox() called with multiple threads in process gpu-process.` makes me think this might be related to recent issues of electron apps not working without the option `--no-sandbox` (I tried the option, and it didn't help, but still).

**Comment by jokuNimi** [comment]
I checked the terminal output when running from appimage version 1.1.7 (which doesn't have the issue):
```
~ % ./sleek-1.1.7.AppImage 
Persisted data located at: /home/valtteri/.config/sleek/user-preferences.json
Checking for update
Success: Translations loaded for: en
libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
[25186:0405/143258.175574:ERROR:sandbox_linux.cc(376)] InitializeSandbox() called with multiple threads in process gpu-process.
Success: Window events setup
Success: Renderer window created
Success: drawerWidth persisted, value is: 500
Success: filterDrawer persisted, value is: false
Success: viewDrawer persisted, value is: false
Success: uid persisted, value is: e7jufsk9wwl
Update for version 1.1.7 is not available (latest version: 1.1.7, downgrade is disallowed).
No update
checkForUpdatesAndNotify called, downloadPromise is null
Success: dismissedMessages persisted, value is: logging
Success: dismissedMessages persisted, value is: logging,share

```
So the `libva` and `InitializeSandox()` errors don't seem to be relevant.

**Comment by ransome1** [comment]
@jokuNimi and you say the app just shows a blank screen or is there anything to see? Do you at least see the top menu which gives you the `About` option? If so you can enable the developer tools in there and hopefully will find some error in the console, that points us to the right direction.

**Comment by ransome1** [comment]
What happens when you press the Alt/Option key? Does the menu show up then?

**Comment by jokuNimi** [comment]
I see nothing happening when pressing alt, ctrl, shift, right clicks and left clicks in various combinations.

**Comment by ransome1** [comment]
@jokuNimi it will probably not solve the issue, but could you please try to uninstall and install sleek again? It has been bumped to v1.1.8 and hopefully this either works or gives us more detail on what is happening on your side.

**Comment by wedeluxe** [comment]
OS: Arch Linux

I ran into the very same issue a couple of days ago.
Until then, sleek 1.1.7 worked like a charm.
After upgrading a couple of other packages, sleek 1.1.7 failed with a white/blank screen.

Package manager shows that sleek depends on the electron and nodejs system packages.
Amongst the package upgrades mentioned above:

electron 17.2.0-1 -> 18.0.1-1

This might be the culprit.

**Comment by wedeluxe** [comment]
Forgot to mention:
sleek 1.1.8 fails in the same way. Shows a blank/white screen.

**Comment by jokuNimi** [comment]
@ransome1 Uninstalled and reinstalled, didn't help.

@wedeluxe Thank you! You got it! I downgraded to electron 17.2.0-1 and rebooted, and sleek worked as it should.  The upgraded back to 18.0.1-1 and get a blank screen again.

FYI there's a package called electron17 on my system, maybe using that would be a low-effort fix?

**Comment by aleksfadini** [comment]
Having the same issue on arch linux. Downgrading electron works, but it's not a workaround that is sustainable for me, sadly

**Comment by YMan84** [comment]
I'm having the same issue as well on my system with Arch Linux, with Electron 18.0.3-2

**Comment by gentlecolts** [comment]
Same issue, Manjaro, haven't been able to use sleek for a while.  Not sure how much it matters, but when run from terminal, my log terminates sooner than the original poster's:

```
Persisted data located at: /home/<user>/.config/sleek/user-preferences.json
Success: Translations loaded for: en
Success: Window events setup
Success: Renderer window created
[1716544:0503/103011.607773:ERROR:sandbox_linux.cc(377)] InitializeSandbox() called with multiple threads in process gpu-process.
```

**Comment by ChongChengAC** [comment]
I had the some question in Manjaro after updating electron to 18.0.1 weeks ago. And i find that the question can be fixed by updating electron from to 18.2.1.

**Comment by vlipiainen** [comment]
I can confirm that the bug is gone on my Arch Linux, using electron version 18.2.1-1. I used sleek from the AppImage for a while, so I'm not sure which exact upgrade fixed it.
